### PR TITLE
convert int8

### PR DESCRIPTION
**Pull request by MarisaKirisame**
@slyubomirsky

### CODE REVIEW

File: aot/aot.py [hunk 1]
```diff
@@ -196,7 +196,7 @@ def lib_and_func_name(name):
     _LIB_COUNTER += 1
     return lib_name, packed_name
 
-def _mk_wrapper(fn, ctx, params):
+def _mk_wrapper(fn, ctx, constants):
     def _wrapper(*args):
         new_params = [convert(a, ctx) for a in params]
```
Comment (slyubomirsky) [inline aot/aot.py]: Forgot to rename?

Comment (slyubomirsky) [reply aot/aot.py]: Kind of weird if the tests still pass this way (do they?)

**Comment by slyubomirsky** [comment]
Other than the forgotten variable renaming, seems alright

**Comment by MarisaKirisame** [comment]
@slyubomirsky thx for reviewing!

**Pull request merged**